src/logout/components/Logout/index.js

```diff
@@ -1,16 +1,20 @@
 import { compose } from 'recompose';
 import { withActions } from 'spunky';
-
+import { connect } from 'react-redux';
 import accountActions from 'shared/actions/accountActions';
-
 import Logout from './Logout';
 import withLogout from '../../hocs/withLogout';
 
+import { emptyAll } from '../../../browser/actions/requestsActions';
+
 const mapActionsToProps = ({ reset }) => ({
   logout: reset
 });
 
 export default compose(
   withActions(accountActions, mapActionsToProps),
+  connect(null, (dispatch) => ({
+    emptyAllRequests: () => dispatch(emptyAll())
+  })),
```

Comment:
In conjunction with my previous comment, this `connect` call would need to shift up above `withActions` so that we can reference the `emptyAllRequests` function from within the new `logout` prop.  Something like this:

```js
const mapDispatchToProps = (dispatch) => ({
  emptyAllRequest: () => dispatch(emptyAll())
});

const mapActionsToProps = ({ reset }, { emptyAllRequests }) => ({
  logout: () => {
    reset();
    emptyAllRequests();
  }
});

export default compose(
  connect(null, mapDispatchToProps),
  withAction(accountActions, mapActionsToProps),
  /* ...snip... */
)(Logout);
```

Reply:
👍

Reply:
I just dig deeper, I don't think what you are trying to do will work?
- withAction works with only one actions from saga and `mapActionsToProps` https://github.com/neoverse/spunky/blob/master/src/hocs/withActions.js
if you really want only one prop which is logout, maybe compose will be a solution

Reply:
Last piece of feedback.  To be consistent with our other files, can this function be lifted into a `mapDispatchToProps` function before it's used here?